module-services/service-evtmgr/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_link_libraries(${PROJECT_NAME}
         sml::sml
         Microsoft.GSL::GSL
         eventstore
+        evtmgr
```

Comment:
Please, do not link product-specific targets(Pure/Harmony) to product-agnostic targets.

Reply:
done